manager/integration/tests/test_system_backup_restore.py

```diff
@@ -112,6 +117,66 @@ def test_system_backup_and_restore_volume_with_data(client, volume_name, set_ran
 
     check_volume_data(restored_volume, data)
 
+@pytest.mark.system_backup_restore   # NOQA
+def test_system_backup_and_restore_volume_with_backingimage(client, volume_name, set_random_backupstore):  # NOQA
+    """
+    Scenario: test system backup and restore volume with backingimage
+
+    Issue: https://github.com/longhorn/longhorn/issues/5085
+
+    Given a backingimage
+    And a volume created with the backingimage
+    When system backup created
+    Then system backup in state Ready
+
+    When volume deleted
+    And backingimage deleted
+    And restore system backup
+
+    When restore system backup
```

Comment:
`When` should be followed by `And` or `Then`.
```suggestion
    And restore system backup
```

Reply:
Fixed